tools/buildtools/ambertools/parmconv.xml

```diff
@@ -0,0 +1,197 @@
+<tool id="parmconv" name="Convert Parameters" version="@VERSION@">
+  <description> to AMBER prmtop in preparation for MMPBSA</description>
+  <macros>
+    <import>macros.xml</import>
+  </macros>
+  <expand macro="requirements">
+    <requirement type="package" version="3.2.0">parmed</requirement>
+    <requirement type="package" version="2019.1">gromacs</requirement>
+    <requirement type="package" version="2.11.1">jinja2</requirement>
+  </expand>
+  <command detect_errors="exit_code">
+    <![CDATA[
+   bash '$bash_script' \$CONDA_PREFIX  &&
+   python '$templating_script' '$inputs' &&
+   export AMBERHOME=\$CONDA_PREFIX &&
+   parmed -i ligand.script -O &&
+   parmed -i receptor.script -O &&
+   parmed -i complex.script -O &&
+   parmed -i solvatedcomplex.script -O
+
+      ]]>
+  </command>
+  <configfiles>
+    <inputs name="inputs"/>
+    <configfile name="bash_script">
+      <![CDATA[
+      for i in "$1/share/gromacs/top/*.ff"; do ln -s \$i -f; done
```

Comment:
should that link into a separate subfolder?

Reply:
I think so parmed needs these *.ff to be in a local directory when converting gromacs topologies. Possibly I could set an environment variable. Will check.

Reply:
I thought parmed was able to find these files in the `share/gromacs` dir

Reply:
I can use a variable - $GMXDATA Thank goodness!

Reply:
@simonbray it is supposed. Maybe updated in a new conda package. Set the GMXDATA var and it worked.